node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/docker/DockerOperationsImpl.java

```diff
@@ -111,15 +110,6 @@ public void createContainer(NodeAgentContext context, ContainerData containerDat
 
         addMounts(context, command);
 
-        // TODO: Enforce disk constraints
```

Comment:
This was removed after 7.67, but was reverted due to Vespa 6 apps: https://github.com/vespa-engine/vespa/pull/10042